lib/vmdb/plugins.rb

```diff
@@ -27,10 +27,15 @@ def each(&block)
     end
 
     def init
+      version_check
```

Comment:
This should probably come *after* loading inflections and registering models because the method itself might need those features.